keyboards/zlabkeeb/vrynboard/keyboard.json

```diff
@@ -0,0 +1,99 @@
+{
+    "manufacturer": "zlabkeeb",
+    "keyboard_name": "VRYNBOARD",
+    "maintainer": "zlabkeeb",
+    "bootloader": "rp2040",
+    "build": {
+        "lto": true
+    },
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true,
+        "oled": true,
+        "rgb_matrix": true
+    },
+    "matrix_pins": {
+        "cols": ["GP13", "GP11", "GP9"],
+        "rows": ["GP14", "GP12", "GP10", "GP8"]
+    },
+    "rgb_matrix": {
+        "animations": {
+            "alphas_mods": true,
+            "gradient_up_down": true,
+            "breathing": true,
+            "band_sat": true,
+            "band_val": true,
+            "band_pinwheel_sat": true,
+            "band_pinwheel_val": true,
+            "cycle_all": true,
+            "cycle_up_down": true,
+            "cycle_out_in": true,
+            "cycle_pinwheel": true,
+            "dual_beacon": true,
+            "rainbow_pinwheels": true,
+            "raindrops": true,
+            "jellybean_raindrops": true,
+            "hue_breathing": true,
+            "hue_pendulum": true,
+            "hue_wave": true,
+            "pixel_fractal": true,
+            "pixel_flow": true,
+            "pixel_rain": true,
+            "solid_reactive_simple": true,
+            "solid_reactive": true,
+            "solid_reactive_multiwide": true,
+            "solid_reactive_multicross": true,
+            "solid_reactive_multinexus": true,
+            "multisplash": true,
+            "solid_multisplash": true
+        },
+        "driver": "ws2812",
+        "layout": [
+            {"matrix": [0, 0], "x": 0, "y": 0, "flags": 1},
+            {"matrix": [1, 0], "x": 0, "y": 16, "flags": 1},
+            {"matrix": [1, 1], "x": 25, "y": 16, "flags": 4},
+            {"matrix": [1, 2], "x": 50, "y": 16, "flags": 1},
+            {"matrix": [2, 0], "x": 0, "y": 32, "flags": 1},
+            {"matrix": [2, 1], "x": 25, "y": 32, "flags": 4},
+            {"matrix": [2, 2], "x": 50, "y": 32, "flags":1},
+            {"matrix": [3, 0], "x": 0, "y": 48, "flags": 1},
+            {"matrix": [3, 1], "x": 25, "y": 48, "flags": 4},
+            {"matrix": [3, 2], "x": 50, "y": 48, "flags": 1}
```

Comment:
Positions should be from 0..224 on the X axis, and 0..64 on the Y axis.

Reply:
![image](https://github.com/user-attachments/assets/61ed41d9-a64e-4d09-aeb1-572b53992a84)
Looks good, but I think all the flags should be set to 4.

Reply:
Thank you very much @fauxpark. 
Okay, I will set the flags to 4